sql_generators/search/__init__.py

```diff
@@ -18,15 +18,28 @@
 from bigquery_etl.format_sql.formatter import reformat
 
 # fmt: off
-APP_CHANNEL_TUPLES = [
-    ("org_mozilla_fenix",           "Firefox Preview",  "beta",      "android"),  # noqa E241 E501
-    ("org_mozilla_fenix_nightly",   "Firefox Preview",  "nightly",   "android"),  # noqa E241 E501
-    ("org_mozilla_fennec_aurora",   "Fenix",            "nightly",   "android"),  # noqa E241 E501
-    ("org_mozilla_firefox_beta",    "Fenix",            "beta",      "android"),  # noqa E241 E501
-    ("org_mozilla_firefox",         "Fenix",            "release",   "android"),  # noqa E241 E501
-    ("org_mozilla_ios_firefox",     "Fennec",           "release",   "ios"),  # noqa E241 E501
-    ("org_mozilla_ios_firefoxbeta", "Fennec",           "beta",      "ios"),  # noqa E241 E501
-    ("org_mozilla_ios_fennec",      "Fennec",           "nightly",   "ios"),  # noqa E241 E501
+FIREFOX_ANDROID_TUPLES = [
+    ("org_mozilla_fenix",           "Firefox Preview",  "beta"),  # noqa E241 E501
+    ("org_mozilla_fenix_nightly",   "Firefox Preview",  "nightly"),  # noqa E241 E501
+    ("org_mozilla_fennec_aurora",   "Fenix",            "nightly"),  # noqa E241 E501
+    ("org_mozilla_firefox_beta",    "Fenix",            "beta"),  # noqa E241 E501
+    ("org_mozilla_firefox",         "Fenix",            "release"),  # noqa E241 E501
+]
+
+FIREFOX_IOS_TUPLES = [
+    ("org_mozilla_ios_firefox",     "Fennec", "release"),  # noqa E241 E501
+    ("org_mozilla_ios_firefoxbeta", "Fennec", "beta"),  # noqa E241 E501
+    ("org_mozilla_ios_fennec",      "Fennec", "nightly"),  # noqa E241 E501
+]
+
+FOCUS_ANDROID_TUPLES = [
+    ("org_mozilla_focus",           "Focus Android Baseline",    "release"),  # noqa E241 E501
```

Comment:
Let's stick with calling them "Glean" and change the "Baseline" ones we have in other tables in a follow-up PR (we'll need to make sure not to break existing dashboards etc here though)
```suggestion
    ("org_mozilla_focus",           "Focus Android Glean",    "release"),  # noqa E241 E501
```